src/routes/bucket/createBucket.ts

```diff
@@ -45,26 +45,18 @@ export default async function routes(fastify: FastifyInstance) {
         owner = await getOwner(jwt)
       } catch (err) {
         console.log(err)
-        return response.status(400).send({
-          statusCode: '400',
-          error: err.message,
-          message: err.message,
-        })
+        return response.status(400).send()
```

Comment:
@ftonato can you add back the error bodies here?

Reply:
Sorry for the delay, I saw that you already corrected it, thanks =)